sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/GalleryTests.cs

```diff
@@ -832,6 +842,16 @@ private void ValidateGalleryApplicationVersion(GalleryApplicationVersion applica
             Assert.NotNull(applicationVersionOut.PublishingProfile.EndOfLifeDate);
             Assert.NotNull(applicationVersionOut.PublishingProfile.PublishedDate);
             Assert.NotNull(applicationVersionOut.Id);
+            Assert.Equal(applicationVersionIn.PublishingProfile.Settings.PackageFileName, applicationVersionOut.PublishingProfile.Settings.PackageFileName);
```

Comment:
Not sure which part caused this, but the Credential scanner PR check is throwing errors in this PR.

Reply:
removed keys in the recorded tests